app/src/main/res/layout/activity_hero_info.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context="com.hyperion.ths.marvel_03.ui.heroinfo.HeroInfoActivity"
+    tools:ignore="ContentDescription|RtlSymmetry"
+    >
+    <data>
+        <variable
+            name="viewModel"
+            type="com.hyperion.ths.marvel_03.ui.heroinfo.HeroInfoViewModel"
+            />
+    </data>
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+
+        >
+        <ImageView
+            android:layout_width="@dimen/dp200"
+            android:layout_height="@dimen/dp200"
+            android:layout_gravity="center"
+            android:layout_marginBottom="@dimen/dp30"
+            android:layout_marginTop="@dimen/dp30"
+            app:imageUrl="@{viewModel.heroImageUrl}"
+            />
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:paddingLeft="@dimen/dp15"
+            android:paddingStart="@dimen/dp15"
+            android:text="@{viewModel.heroId}"
+            android:textSize="@dimen/sp_17"
+            android:textStyle="bold"
+            />
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:paddingLeft="@dimen/dp15"
+            android:paddingStart="@dimen/dp15"
+            android:text="@{viewModel.heroName}"
+            android:textSize="@dimen/sp_17"
+            />
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:paddingLeft="@dimen/dp15"
+            android:paddingStart="@dimen/dp15"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingStart` you should probably also define `paddingEnd` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.